### DISCUSSION TITLE
Occasionally getting SIGABRT from dev server

### DISCUSSION THREAD

**Discussion by niwsa** [vercel/next.js#13158] [Help]
Hi anyone encountering crashes while running dev server like this

>Assertion failed: (0), function uv_close, file ../deps/uv/src/unix/core.c, line 178.

From discussion in https://github.com/webpack/webpack/issues/10037#issuecomment-630367726 it appears to be something with respect to file watcher watchpack..

next version 9.4.1

**Comment by devth** [comment]
Yes, I'm seeing this more often. `    "next": "^9.5.3",`

**Reply by niwsa** [reply]
@devth Its been a while since I encountered that error,also it occured when I was getting started with next.js might have done something silly at that time...suffice to say that haven't encountered since then...also until 5 days back I used v9.5.3 and now I am on 9.5.5.
